login/routers/middlewares/SignUpValidator.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+
+const router = express.Router();
+const RegistrationRequest = require('../../db/db_objects/RegestrationRequest.js');
+const signUpValidation = require('../../core/validations/SignUpValidation.js');
+
+router.post('/', (request, response,next) => {
+    const registrationRequest = new RegistrationRequest(request.body.login, request.body.email, request.body.password);
+    if(signUpValidation.SignUpValidation(registrationRequest, response)){
```

Comment:
nit: btw, what's the naming convention for functions in node.js? I thought it was lowercase, could be wrong though.